packages/guides/src/Nodes/SectionNode.php

```diff
@@ -5,15 +5,28 @@
 namespace phpDocumentor\Guides\Nodes;
 
 use function array_merge;
+use function array_unique;
+use function trim;
 
 /** @extends CompoundNode<Node> */
-final class SectionNode extends CompoundNode
+final class SectionNode extends CompoundNode implements TitledNode
 {
+    private string $identifier;
+    /** @var string[] */
+    private array $anchors = [];
+
     public function __construct(private readonly TitleNode $title)
     {
+        $this->identifier = $title->getId();
```

Comment:
Why would you store this in a property? GetIdentifier could simply proxy to title?